### ISSUE TITLE
UI doesn't support URLs that are IP address-based

### ISSUE THREAD

**Issue by douglas-andrew-harley** [fhir-crucible/crucible#134]
When you attempt to provide a FHIR server URL using an IP address, the UI refuses to accept it, showing this error: "Please enter a valid URL."

WORKAROUND: create a fake domain mapping in the Crucible server's /etc/hosts file to point to the desired IP address.

**Comment by okeefm** [comment]
Is the IP address you're trying to use a [reserved private IP](https://en.wikipedia.org/wiki/Private_network#Private_IPv4_address_spaces), such as `192.168.1.100`? If not, appending an `http://` to the front of the IP should fix the issue. If it is a private IP, that is an issue we've seen before; we'll prioritize looking into the issue in the near term.

**Comment by douglas-andrew-harley** [comment]
Yes, it is indeed a reserved private IP address.  Cool, thanks for the info...I am good-to-go with my workaround though, so no rush, I just wanted to bring it to your attention.

Background: I am running HAPI-FHIR server on my windows laptop, and Crucible-FHIR on an Ubuntu VM hosted within my laptop, so private IP address was the only solution I could come up with.

**Comment by arscan** [comment]
@okeefm and @andrequina,has this been resolved? I believe that this was fixedin PR https://github.com/fhir-crucible/crucible/pull/160

I tried it out on my local machine and I was able to put in `http://192.168.0.1:3000`, so I think that this should be all set.  By default we allow reserved private ip addresses when in development, but restrict them when in production.  There is a variable in `application.rb` now that lets you override that behavior.
